packages/gatsby-theme/src/gatsby-plugin-theme-ui/index.js

```diff
@@ -1,4 +1,106 @@
+const bColors = {
+  transparent: "transparent",
+  black: "#000",
+  white: "#fff",
+  primary: "#0CBFC7",
+  secondary: "#CA3E73",
+  darkShade: "#37375B",
+  lightShade: "#F7F7F7",
+}
+
+const bFonts = {
+  sans:
+    '"Noto Sans", -apple-system, BlinkMacSystemFont,"Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji"',
+  headings: "'Poppins', sans-serif",
+}
+
+const bFontSizes = {
+  tiny: ".875rem",
+  base: "1rem",
+  lg: "1.125rem",
+  xl: "1.25rem",
+  "2xl": "1.5rem",
+  "3xl": "2rem",
+  "4xl": "2.25rem",
+  "5xl": "3rem",
+}
+
+const bLineHeights = {
+  none: "1",
+}
+
+const bHeadings = {
+  base: {
+    fontFamily: "heading",
+    fontWeight: "heading",
+    lineHeight: "heading",
+    my: "xsmall",
+  },
+  intro: {
+    textTransform: "uppercase",
+    mb: ["xsmall", null, "small"],
+    mt: "xsmall",
+    fontWeight: "normal",
+    lineHeight: "normal",
+  },
+}
+const bShadows = {
+  default: "0 1px 3px 0 rgba(0, 0, 0, 0.1), 0 1px 2px 0 rgba(0, 0, 0, 0.06)",
```

Comment:
There is another shadow declaration that is currently being used in the Button component. We should add it.
https://github.com/octahedroid/drupal-gatsby-theme/blob/ad1ca942d8e864c2245f1947ad152d7b3b4e53c4/packages/gatsby-theme/src/components/Button/index.js#L29

Reply:
Perfect! I'm going to add it. Thanks!